verified-broadcast/README.md

```diff
@@ -10,36 +10,37 @@ This project requires some environment variables to be set. To keep your tokens
-| TWILIO_PHONE_NUMBER | The Twilio phone number to send broadcast SMS from | Yes |
-| PASSCODE | Choose a passcode for your app. Users have to use this passcode to send SMS broadcasts or delete all subscriptions | Yes |
-| BROADCAST_NOTIFY_SERVICE_SID       | SID of your Twilio Notify Service. Otherwise one will automatically be provisioned for you. | No      |
-| MESSAGING_SERVICE_SID | SID of a Twilio Messaging Service. Otherwise one will automatically be provisioned for you. | No |
-| VERIFY_SERVICE_SID           | SID of your Twilio Verify Service. Create one [here](https://www.twilio.com/console/verify/services). Otherwise one will automatically be provisioned for you. | No      |
+| Variable                     | Description                                                                                                                                                    | Required |
+| :--------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| TWILIO_PHONE_NUMBER          | The Twilio phone number to send broadcast SMS from                                                                                                             | Yes      |
+| AUTH_USERNAME                | Choose a passcode for your app. Users have to use this username and passcode to send SMS broadcasts or delete all subscriptions                                | Yes      |
+| AUTH_PASSCODE                | Choose a passcode for your app. Users have to use this username and passcode to send SMS broadcasts or delete all subscriptions                                | Yes      |
+| BROADCAST_NOTIFY_SERVICE_SID | SID of your Twilio Notify Service. Otherwise one will automatically be provisioned for you.                                                                    | No       |
+| MESSAGING_SERVICE_SID        | SID of a Twilio Messaging Service. Otherwise one will automatically be provisioned for you.                                                                    | No       |
+| VERIFY_SERVICE_SID           | SID of your Twilio Verify Service. Create one [here](https://www.twilio.com/console/verify/services). Otherwise one will automatically be provisioned for you. | No       |
 
 ### Function Parameters
 
 `start-verify.js` expects the following parameters:
 
-| Parameter      | Description                                 | Required |
-| :------------- | :------------------------------------------ | :------- |
-| `to`       | Subscriber phone number. | Yes  |
+| Parameter | Description              | Required |
+| :-------- | :----------------------- | :------- |
+| `to`      | Subscriber phone number. | Yes      |
 
 `broadcast.js` has to be authenticated using HTTP Basic Auth using `admin` as username an the configured `PASSCODE` as password. It expects the following parameters:
```

Comment:
This needs updating to `AUTH_USERNAME` and `AUTH_PASSCODE` too.

Reply:
Ahh yep, I missed that instance along with one later in the README! It should hopefully be good to go now.

Reply:
Looking good now!